pages/ListPage.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import CampingCard from "../components/CampingCard";
+import { campingTrips } from "../data/campingTrips";
+import { Container, Card, Header, Footer } from "../styles/styles";
+
+export default function ListPage() {
+  return (
+    <>
+      <Header>
+        <h1>Organize your Trips</h1>
+      </Header>
+      <Container>
+        {campingTrips.map((trip) => (
+          <Card key={trip.id}>
+            <CampingCard trip={trip} />
+          </Card>
```

Comment:
I don't understand why you have the Card-Component here at this point and then again inside the campingCard.js